webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/NodeApi.scala

```diff
@@ -467,11 +470,34 @@ class NodeApiService13 (
   , readOnlySoftwareDAO: ReadOnlySoftwareDAO
   , restExtractor: RestExtractorService
   , getGlobalMode : () => Box[GlobalPolicyMode]
+  , reportingService: ReportingService
 ) extends Loggable {
 
-  def serialize(agentRunWithNodeConfig: Option[AgentRunWithNodeConfig], globalPolicyMode: GlobalPolicyMode, nodeInfo : NodeInfo, properties : List[String], softs: List[Software]) = {
+  def serialize(agentRunWithNodeConfig: Option[AgentRunWithNodeConfig], globalPolicyMode: GlobalPolicyMode, nodeInfo : NodeInfo, properties : List[String], softs: List[Software], compliance : Option[NodeStatusReport], sysCompliance : Option[NodeStatusReport]) = {
     import net.liftweb.json.JsonDSL._
 
+
+      
+    def toComplianceArray(comp : ComplianceLevel) : JArray =
+      JArray (
```

Comment:
I thought we didn't do that anymore because of rounding errors, and that we were only using number of components / total number ?

Reply:
I send the same data we used to send

Reply:
ok, strange, I must be mistaken.